### PR TITLE
Chore: Update commands.json to expand labels and their related projects

### PR DESCRIPTION
**Pull request by academo**
**What is this feature?**

Updates our existing commands.json that acts when labels are added to issues.

This update adds a more comprehensive list of labels that were not previously included in the commands.json file.

**Why do we need this feature?**

We are moving towards an automated triaging of issues and we need to ensure the labels we'll use to auto-triage
will assign the issue to the correct project board.

**Who is this feature for?**

All issues opened in the Grafana repository.

**Where does the list of labels and projects come from?**

The list of labels was gathered from the current list of github labels https://github.com/grafana/grafana/labels?page=20&sort=count-desc and then matched (by hand) to the github projects based on the internal teams responsibilities document.

It is much possible some of the labels have an incorrect project assigned as the work was done in a "good faith effort"

## Big thanks to @tonypowa and @usmangt for going through all the list of labels!

### CODE REVIEW

File: .github/commands.json [hunk 1]
```diff
@@ -458,5 +458,637 @@
     "addToProject": {
       "url": "https://github.com/orgs/grafana/projects/471"
     }
+  },
+  {
+    "type": "label",
+    "name": "area/alerting",
+    "action": "addToProject",
+    "addToProject": {
+      "url": "https://github.com/orgs/grafana/projects/52/"
```
Comment (tolzhabayev) [inline .github/commands.json:467]:
```suggestion
      "url": "https://github.com/orgs/grafana/projects/52"
```

File: .github/commands.json [hunk 2]
```diff
@@ -458,5 +458,637 @@
...
+    "addToProject": {
+      "url": "https://github.com/orgs/grafana/projects/482"
+    }
+  },
+  {
+    "type": "label",
+    "name": "area/backend/db/postgres",
+    "action": "addToProject",
+    "addToProject": {
+      "url": "https://github.com/orgs/grafana/projects/96/"
```
Comment (tolzhabayev) [inline .github/commands.json:811]:
```suggestion
      "url": "https://github.com/orgs/grafana/projects/96"
```

File: .github/commands.json [hunk 3]
```diff
@@ -458,5 +458,637 @@
...
+    "addToProject": {
+      "url": "https://github.com/orgs/grafana/projects/457"
+    }
+  },
+  {
+    "type": "label",
+    "name": "datasource/TestDataDB",
+    "action": "addToProject",
+    "addToProject": {
+      "url": "https://github.com/orgs/grafana/projects/76/"
```
Comment (tolzhabayev) [inline .github/commands.json:947]:
```suggestion
      "url": "https://github.com/orgs/grafana/projects/76"
```

File: .github/commands.json [hunk 4]
```diff
@@ -458,5 +458,637 @@
...
+    "addToProject": {
+      "url": "https://github.com/orgs/grafana/projects/457"
+    }
+  },
+  {
+    "type": "label",
+    "name": "datasource/Alertmanager",
+    "action": "addToProject",
+    "addToProject": {
+      "url": "https://github.com/orgs/grafana/projects/52/"
```
Comment (tolzhabayev) [inline .github/commands.json:995]:
```suggestion
      "url": "https://github.com/orgs/grafana/projects/52"
```

**Review by tolzhabayev — COMMENTED** [review]
Fix the trailing slash'es then we can get this merged. We can always adjust the single cases if we see them ending up being wrong.

**Review by moxious — COMMENTED** [review]
No changes beyond what Timur already caught.  Thank you!  Great improvement!

**Review by tolzhabayev — APPROVED** [review]

**Pull request merged**